README.md

```diff
@@ -1,703 +1,48 @@
-<p align="center">
-  <img src="https://github.com/lucasbento/create-graphql/raw/master/content/logo.png">
-</p>
+# ![Create-GraphQL Logo](/media/logo.png)
 
-<h1 align="center">Create GraphQL</h1>
+# Create-GraphQL
```

Comment:
I would rather use the HTML tags on the title and the badges because we can center them on the README.